tools/github/data/common-labels.csv

```diff
@@ -120,6 +120,7 @@ Devtestlab,,e99695
 Digital Twins,,e99695
 Do Not Merge,,b60205
 Docs,,e99695
+Durable Task Scheduler,,e99695
```

Comment:
We'll need to remember to remove "Durable Task" from the .NET repository as well.